packages/react-components/react-jsx-runtime/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { jsx } from './jsx';
```

Comment:
Should it export `jsxs`?

Reply:
For the case of using jsx factory, yes. For the case of using import source, doesn't matter. So in case of doubt, we can support both scenarios by exporting